src/main.rs

```diff
@@ -128,7 +128,7 @@ fn main() -> Result<(), Box<dyn Error>> {
             }
         }
     }
-    let mut last_time = None;
+    let mut last_time: std::option::Option<chrono::DateTime<chrono::FixedOffset>> = None;
```

Comment:
Why does type inference fail here? And why are you spelling out `Option`?

Reply:
When trying to make it compile, the compiler was confused about this variable for a moment.  I decided that making it compile would be more important than relying at most on type inference.

I spelled it out due to the compiler's error message.  But I can revert that.  :wrench:

Reply:
What do you mean by "for a moment"?

Reply:
When trying to move the basic logic to dedicated files therefore, I had at lot to fix regarding the scopes and so on.  The compiler was able to help with that.

"But when suddenly this weird `src/lib.rs` appeared without any previous warning," the compiler might have thought, "I had no idea where all the logic moved to.  And without this type annotation, it looked even more confusing."

Basically, the fields all became private and had to be fixed with public getters.  No problem, basically.  But I did not found all occurrences at the first run of `cargo check`, so there were a lot of complaints.  One was exactly about the missing type.  I added it to make it compile and left it in, just to be sure.

But I can revert it. :wrench:

Reply:
:wrench: :heavy_check_mark: